scripts/bulk-access-control.md

```diff
@@ -0,0 +1,93 @@
+# Bulk Access Control Management script
+[Back to the list of all defined scripts](./index.md)
+
+<!-- TOC -->
+* [Import script](#bulk-access-control-management-script)
```

Comment:
This says "Import Script" instead of the script name.

Reply:
fixed